src/pop/storage/util.cpp

```diff
@@ -126,16 +116,7 @@ bool loadTrees(AltBlockTree& tree,
   }
   std::vector<typename VbkBlockTree::stored_index_t> vbkblocks;
   typename VbkBlock::hash_t vbktip;
-  if (!detail::loadBlocksAndTip(
-          vbkblocks,
-          vbktip,
-          storage,
-          state,  // on every block, take its hash and warmup progpow header
-                  // cache
-          [](VbkBlock::hash_t hash, const StoredBlockIndex<VbkBlock>& index) {
-            auto serializedHeader = SerializeToRaw(*index.header);
-            progpow::insertHeaderCacheEntry(serializedHeader, std::move(hash));
-          })) {
```

Comment:
Why is it removed? This code should provide quite a speedup for the VBK hash calculation.